weaver/wps_restapi/templates/notification_email_example.mako

```diff
@@ -1,3 +1,4 @@
+## -*- coding: utf-8 -*-
```

Comment:
Does the double # make a difference?

Reply:
I copied from the documentation... But something as simple as ‘#coding: utf8’ would work I think.